marketplacetests/marketplace/pages/app_details.py

```diff
@@ -34,6 +35,13 @@ def first_review_body(self):
     def first_review_rating(self):
         return int(self.marionette.find_element(*self._first_review_locator).get_attribute('data-rating'))
 
+    @property
+    def open_website_href(self):
```

Comment:
This sounds like it would open the website, maybe drop `open_` from the method name?